src/main/java/com/github/wovnio/wovnjava/Api.java

```diff
@@ -50,12 +51,16 @@ String translate(String lang, String html) throws ApiException {
             con.setReadTimeout(settings.readTimeout);
             return translate(lang, html, con);
         } catch (UnsupportedEncodingException e) {
+            WovnLogger.log("API error: unsupported encoding.");
             throw new ApiException("UnsupportedEncodingException", e.getMessage());
         } catch (MalformedURLException e) {
+            WovnLogger.log("API error: malformed URL.");
             throw new ApiException("MalformedURLException", e.getMessage());
         } catch (IOException e) {
+            WovnLogger.log("API error: IO exception.");
```

Comment:
Losing information from the exception here. Does passing in `e` to the log message include the original exception message?

Reply:
I didn't include the exception because they are thrown right after that and a stack trace would be in the log. 
If e is included, it will added a stack trace right after the error message.
Now I think about it I don't even think these log messages are needed, because the stack trace would be logged anyway.